src/styles/theme.js

```diff
@@ -0,0 +1,95 @@
+import { css } from "styled-components";
+
+const breakpoints = {
+  tablet: 1040,
+  mobile: 640
+};
+
+const media = Object.keys(breakpoints).reduce((acc, label) => {
+  acc[label] = (...args) => css`
+    @media screen and (max-width: ${breakpoints[label] / 16}em) {
+      ${css(...args)}
+    }
+  `;
+
+  return acc;
+}, {});
+
+const theme = {
+  color: {
+    accent: {
+      primary: "#226ce7",
+      secondary: "#f42e4f"
+    },
+    background: {
+      base: "#fff",
+      light: "#f5f5f5"
+    },
+    text: {
+      primary: "#222",
+      base: "#666",
+      light: "#aaa",
+      lighter: "#fff",
+      money: "#3ccb3c"
+    },
+    elements: {
+      icon: "#ddd",
+      border: "#ddd",
+      line: "#eee",
+      menu: "#666",
+      dropdown: "#222",
+      tickmark: "#3ccb3c",
+    },
+  },
+  font: {
+    family: {
+      base: "\"Lato\", sans-serif"
+    },
+    size: {
+      base: "14px",
+      heading: {
+        primary: "2rem",
+        secondary: "1.5rem",
+        tertiary: "1.25rem",
+        base: "1rem"
+      },
+      label: {
+        primary: "1.25rem",
+        secondary: "0.875rem",
+        tertiary: "0.75rem",
+        base: "1rem"
+      }
+    },
+    letterSpacing: {
+      wide: "0.1rem"
+    },
+    lineHeight: {
+      heading: "1.3",
+      label: "1.2",
+      paragraph: "1.8"
+    }
+  },
+  size: {
+    border: {
+      base: "2px",
+      thin: "1px"
+    }
+  },
+  effects: {
+    shadow: {
+      box: "0px 3px 15px 0 rgba(0,0,0,.06)",
+      header: "0px 3px 15px 0 rgba(0,0,0,.06)",
```

Comment:
use `rgba(0, 0, 0, 0.06)` // there is no stylelint for styledcomponents

Reply:
Do you mean added spaces and zeros?